lib/widgets/icon.dart

```diff
@@ -95,6 +96,7 @@ class OBIcon extends StatelessWidget {
                 iconData.nativeIcon,
                 size: iconSize,
                 color: iconColor,
+                semanticLabel: semanticLabel ?? 'Close',
```

Comment:
Uhh no.. this way all icons in the app will be labeled as `'Close'`.

Don't set anything by default and whatever used, pls localise.

Reply:
Oh shit, thought i was only setting it for the close button, my bad.